src/js/views/globals/add-patient/add-patient_views.js

```diff
@@ -227,10 +201,6 @@ const AddPatientModal = View.extend({
   onClose() {
     this.destroy();
   },
-  onModelChange() {
```

Comment:
not needed?

Reply:
It was never actually being triggered. I accidentally left it in in the first round of dev.